### DISCUSSION TITLE
Add error message and data why a build was canceled

### DISCUSSION THREAD

**Discussion by hfhbd** [community/community#57145] [Actions]
### Select Topic Area

Product Feedback

### Body

We sometimes hit the limitations of the runners, like using too much cpu. These action runs are simply canceled without any information: `Error: Process completed with exit code 143.` As a user, it is impossible to find out, which limitation was hit, like CPU or RAM limitation. 

Instead, please add some information, why the run was canceled to allow analyzing the root cause from a user.

See also: https://github.com/actions/runner-images/issues/6680

**Comment by mostafaznv** [comment]
how did you fix this issue?
